XmlFileType.js

```diff
@@ -0,0 +1,541 @@
+/*
+ * XmlFileType.js - Represents a collection of XML files
+ *
+ * Copyright © 2021, Box, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var fs = require("fs");
+var path = require("path");
+// var log4js = require("log4js");
+var mm = require("micromatch");
+var XmlFile = require("./XmlFile.js");
+
+// var logger = log4js.getLogger("loctool.plugin.XmlFileType");
```

Comment:
Did you disable logging for `XmlFileType` on purpose?

Reply:
Yes, because of this: https://github.com/iLib-js/loctool/pull/185

Log4js logging didn't work unless you set up a special log4js config file for each plugin, which is a pain in the you-know-what. Now, it can just use the regular log4js that the loctool uses so that its output comes mixed in with the regular loctool output.